src/app.js

```diff
@@ -25,11 +33,64 @@ const quoteData = [
   },
 ];
 
+quoteData.forEach(function(quote) {
+  $('#dropdown').append(`<option>${quote.symbol}</option>`)
+})
+
+
+
 $(document).ready(function() {
+
+
+  let bus = {};
+  bus = _.extend(bus, Backbone.Events);
+
+  let template = _.template($('#quote-template').html());
+  let tradeTemplate = _.template($('#trade-template').html());
+  let orderTemplate = _.template($('#order-template').html());
+
   const quotes = new QuoteList(quoteData);
   const simulator = new Simulator({
     quotes: quotes,
   });
+  const orders = new OrderList();
+
+  const quoteListView = new QuoteListView({
+    el: '#quotes-container',
+    model: quotes,
+    template: template,
+    bus: bus,
+  });
+
+  const tradeListView = new TradeListView({
+    el: '#trades-container',
+    template: tradeTemplate,
+    bus: bus,
+  })
+  const orderFormView = new OrderFormView({
+    // quoteData: quoteData,
+    el: '.order-entry-form',
+    bus: bus,
+    orderList: orders,
+    quoteList: quotes,
+  });
+
+  const order = new Order({
```

Comment:
It doesn't look like this instance of `Order` is being used anywhere, so we could probably remove it?